includes/templates/reader-activation-emails/verification.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/**
+ * Homepage Pattern.
+ *
+ * @package Newspack
+ */
+
+// phpcs:disable WordPressVIPMinimum.Security.Mustache.OutputNotation
+
+return array(
+	'post_title'   => __( 'Your verification link', 'newspack' ),
+	'post_content' => '<!-- wp:site-logo {"align":"center"} /-->
+
+  <!-- wp:heading -->
+  <h2>Hi there!</h2>
+  <!-- /wp:heading -->
+
+  <!-- wp:paragraph -->
+  <p>To manage your account, please verify your email address by clicking the link below:</p>
+  <!-- /wp:paragraph -->
+
+  <!-- wp:buttons -->
+  <div class="wp-block-buttons"><!-- wp:button {"style":{"color":{"background":"#dd3333"}}} -->
+  <div class="wp-block-button"><a class="wp-block-button__link has-background" href="*VERIFICATION_URL*" style="background-color:#dd3333">Verify my email</a></div>
+  <!-- /wp:button --></div>
+  <!-- /wp:buttons -->
+
+  <!-- wp:paragraph {"style":{"typography":{"fontSize":"14px"}},"textColor":"secondary-variation"} -->
```

Comment:
Not sure I like how the font size for this paragraph is slightly smaller than the rest of the email—it creates a weird vertical rhythm that looks unintentional.

<img width="608" alt="Screen Shot 2022-08-24 at 1 24 12 PM" src="https://user-images.githubusercontent.com/2230142/186505553-ddedac26-265c-4d60-9773-684fd7d8b469.png">

Reply:
Updated in bef2cd40ef8c60bda40973b81f8e509da80403d4